Jenkinsfile

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/env groovy
+// vim: set ts=4 sts=4 sw=4 noet:
+
+pipeline {
+	agent any
+	environment {
+		DOCKER_IMG = "docker.lco.global/pupe-pat"
+		GIT_COMMIT_SHORT_ID = getCommit().take(12)
+		GIT_TAG_NAME = gitTagName()
```

Comment:
This can be done using the shared libraries with:
```
PROJ_NAME = projName()
GIT_DESCRIPTION = gitDescription()
DOCKER_IMG = dockerImageName("${LCO_DOCK_REG}", "${PROJ_NAME}", "${GIT_DESCRIPTION}")
```
The variable names used obvisously don't matter.